src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -65,6 +65,37 @@ public virtual void CleanUp()
         /// </summary>
         public abstract IEnumerable<NuGetProject> GetSelectedProjects(UserAction action);
 
+        private int _selectedIndex = 0;
```

Comment:
Use auto property here if you need to expose it publicly.

Reply:
Fixed